### PR TITLE
Create osd-recurring-meeting

### PR DESCRIPTION
**Pull request by simonv3**
This post doesn't have a date because I think it makes sense to list it permanently at the top of our listing. I can do that in another PR.

### CODE REVIEW

File: osd-recurring-online-meeting.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title:  "OSD Monthly Online Get Together"
```
Comment (jancborchardt) [inline osd-recurring-online-meeting.md]: Let’s call it »Open Source Design«, not OSD ;)

File: osd-recurring-online-meeting.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+date:   2016-12-06
+categories: online meet-up
+eventDate: Every First Tuesday
+location: Online at http://talky.io/opensourcedesign
```
Comment (jancborchardt) [inline osd-recurring-online-meeting.md]: Let’s use https here :) below too

**Comment by jancborchardt** [comment]
Apart from my comments it looks good! :)

File: osd-recurring-online-meeting.md (hunk 1 above)
Comment (simonv3) [reply osd-recurring-online-meeting.md]: I was trying to fit it all in the title, afraid open source design would get cut off, but I'll edit :)

**Comment by simonv3** [comment]
@jancborchardt Updated!

**Comment by simonv3** [comment]
visible here: http://opensourcedesign.net/events/open-source-design-monthly-online-meeting/

Also added link here: http://opensourcedesign.net/events/

**Pull request merged**